Automattic/wp-calypso
# Masterbar: Add signup and login links when logged out

This adds links to the masterbar for any logged-out _Calypso_ page, e.g. https://wordpress.com/start or https://wordpress.com/design

**Before**
![screen shot 2016-04-21 at 16 07 47](https://cloud.githubusercontent.com/assets/7767559/14713997/4297057e-07db-11e6-98d6-16a0cf91e07e.png)
![screen shot 2016-04-21 at 16 07 33](https://cloud.githubusercontent.com/assets/7767559/14713999/43c182da-07db-11e6-8ef4-788ee6f91f18.png)


**After**
![screen shot 2016-04-21 at 16 04 37](https://cloud.githubusercontent.com/assets/7767559/14713895/df21ba2a-07da-11e6-984e-69ebdde5893c.png)
![screen shot 2016-04-21 at 16 04 17](https://cloud.githubusercontent.com/assets/7767559/14713903/e4093590-07da-11e6-9cf8-ce38c41dcef7.png)

**To Test**
* Go to http://calypso.localhost:3000/devdocs/start
* The _Sign up_ link should go to /start
* The _Log in_ link should prompt for a login

NOTE: the _Sign up_ link will not work properly from the /design or /theme routes until #4864 is merged.
